channels/ht.m3u

```diff
@@ -3,5 +3,5 @@
 http://66.175.238.147:1935/live/myStream/chunklist_w1749202216.m3u8
 #EXTINF:-1 tvg-id="TeleLouange.ht" tvg-name="Tele Louange" tvg-country="HT" tvg-language="French" tvg-logo="https://i.imgur.com/vZmlq9N.jpg" group-title="",Tele Louange
 https://5790d294af2dc.streamlock.net/8124/8124/chunklist.m3u8
-#EXTINF:-1 tvg-id="TeleHaiti.ht" tvg-name="TeleHaiti" tvg-country="HT" tvg-language="French" tvg-logo="https://www.newsfromhaiti.com/images/TeleHaiti.jpg" group-title="",TeleHaiti (1088p)
+#EXTINF:-1 tvg-id="TeleHaiti.ht" tvg-name="TeleHaiti" tvg-country="HT" tvg-language="French" tvg-logo="https://www.newsfromhaiti.com/images/TeleHaiti.jpg" group-title="News",TeleHaiti (1088p)
```

Comment:
Not quite sure about that tag, chief.
![vlcsnap-2021-07-11-14h02m16s572](https://user-images.githubusercontent.com/30985701/125194225-a1845f00-e250-11eb-9389-d8362a6996c2.png)